pod.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+var (
+	descContainerRestarts = prometheus.NewDesc(
+		"container_restarts",
+		"The number of container restarts per container.",
+		[]string{"namespace", "pod_name", "container_name"}, nil,
```

Comment:
We ran into the discussion of naming here previously. If we stay consistent with the last two PRs, we strip the "_name" suffixes here and just have it as `pod` and `container`. 

The question is basically whether this will collide with labels added further up in the monitoring stack. My hunch is that with "_name" there's about equal likelihood. So as long as we don't make it "kubernetes_pod_name" etc. it probably doesn't matter.

Maybe opt for the simpler version of no suffix and stay consistent for now?